packages/galaxy/rollup.config.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  addPackageFileExports,
+  createRollupConfig,
+} from '@scalar/build-tooling'
+
+const entries = ['src/index.ts']
+
+export default createRollupConfig({
+  typescript: true,
+  copy: [
+    { src: 'src/specifications/3.1.yaml', dest: 'dist' },
+    {
+      src: 'src/specifications/3.1.yaml',
+      dest: 'dist',
+      rename: 'latest.yaml',
+    },
+  ],
+  options: {
+    input: entries,
+  },
+})
+
+await addPackageFileExports({ entries })
```

Comment:
I am having issues with this overwriting the the .json and .yaml file exports in the package.json

Reply:
That is precisely what it does. In your case I would suggest just manually provision the exports field.